docs/src/main/asciidoc/spring-cloud-cloud-foundry-connector.adoc

```diff
@@ -108,7 +108,7 @@ The connector will check for:
 The connector will check for:
 
 * `uri` or `uris` using the scheme `oracle`
-* `jdbcUrl` field in `credentials` using the scheme `oracle`
+* `jdbcUrl` field in `credentials`
```

Comment:
@anwarchk This isn't quite right. It should be 

```
* `jdbcUrl` field in `credentials` using the subprotocol `oracle` in the JDBC URL
```

In other words, the value of the `jdbcUrl` field must start with `jdbc:oracle` in order to get a match.